plugins/Status.cpp

```diff
@@ -87,7 +87,7 @@ bool BedrockPlugin_Status::peekCommand(BedrockNode* node, SQLite& db, BedrockNod
         content["state"] = SQLCStateNames[node->getState()];
         content["hash"] = node->getHash();
         content["commitCount"] = SToStr(node->getCommitCount());
-        content["version"] = SVERSION;
+        content["version"] = _args ? (*_args)["version"] : "";
```

Comment:
We should probably put the plugin name into the version, such that when we sort it there isn't any chance of ambiguity.  For example, imagine plugins A and B, with versions 1 and 2.  And imagine one has A:1 and B:2, while the other has A:2 and B:1.  Without the names, both nodes will have version "1,2", which seems like a match, even though they're different.